avm/res/service-networking/traffic-controller/main.bicep

```
@@ -207,21 +226,41 @@ output associations array = [
   }
 ]
 
+@description('The security policies of the Application Gateway for Containers.')
+output securityPolicies array = [
+  for (securityPolicy, i) in (!empty(securityPolicies) ? array(securityPolicies) : []): {
```

```suggestion
  for (securityPolicy, i) in (securityPolicies ?? []): {
```

Not sure why its written differently for this and other outputs. The above should work just fine